gwml2_schema.sql

```diff
@@ -0,0 +1,206 @@
+);
+
+CREATE TABLE groundwater.borehole_bholeStartPoint (
+	borehole int4 REFERENCES groundwater.borehole(id) ON UPDATE CASCADE ON DELETE CASCADE,
+	bholeStartPoint int4 REFERENCES groundwater.BoreholeStartPointTypeTerm(id) ON UPDATE CASCADE ON DELETE CASCADE
+);
+
+CREATE TABLE groundwater.borehole_bholeCoreInterval (
+	borehole int4 REFERENCES groundwater.borehole(id) ON UPDATE CASCADE ON DELETE CASCADE,
+	bholeCoreInterval int4 REFERENCES groundwater.GM_Envelope(id) ON UPDATE CASCADE ON DELETE CASCADE
+);
+
+CREATE TABLE groundwater.borehole_bholeDriller (
+	borehole int4 REFERENCES groundwater.borehole(id) ON UPDATE CASCADE ON DELETE CASCADE,
+	bholeDriller int4 REFERENCES groundwater.CI_ResponsibleParty(id) ON UPDATE CASCADE ON DELETE CASCADE
+);
+
+CREATE TABLE groundwater.BoreholeDrillingMethodTerm (
+	id serial PRIMARY KEY,
+	prefix VARCHAR (50),
+	p_name VARCHAR (50),
+	uri VARCHAR (255),
+	term_e VARCHAR (150)
+);
+
+CREATE TABLE groundwater.borehole_bholeDrillingMethod (
+	borehole int4 REFERENCES groundwater.borehole(id) ON UPDATE CASCADE ON DELETE CASCADE,
+	bholeDrillingMethod int4 REFERENCES groundwater.BoreholeDrillingMethodTerm(id) ON UPDATE CASCADE ON DELETE CASCADE
+);
+
+CREATE TABLE groundwater.borecollar (
+  	id serial PRIMARY KEY,
+  	collarElevation float(8),
+  	collarElevationType VARCHAR (255),
+  	collarHeadworkType VARCHAR (255),
+  	collarLocation geometry(POINT,4326),
+  	bholeHeadworks int4 REFERENCES groundwater.borehole(id) ON DELETE CASCADE
+);
+
+CREATE TABLE groundwater.Log_Value (
+	id serial PRIMARY KEY,
+	fromDepth int4 REFERENCES groundwater.Quantity(id) ON DELETE SET NULL,
+	toDepth int4 REFERENCES groundwater.Quantity(id) ON DELETE SET NULL
+);
+
+CREATE TABLE groundwater.GW_GeologyLogCoverage (
+	id serial PRIMARY KEY
+);
+
+CREATE TABLE groundwater.gw_geologylogcoverage_logvalue (
+	gw_geologylogcoverage int4 REFERENCES groundwater.GW_GeologyLogCoverage(id) ON UPDATE CASCADE ON DELETE CASCADE,
+	value int4 REFERENCES groundwater.Log_Value(id) ON UPDATE CASCADE ON DELETE CASCADE
+);
+
+CREATE TABLE groundwater.GW_GeologyLog (
+	id serial PRIMARY KEY,
+	phenomenonTime timestamp,
+	resultTime timestamp,
+	parameter  VARCHAR (150),
+	resultQuality int4 REFERENCES groundwater.GW_GeologyLogCoverage(id) ON DELETE SET NULL,
+	startDepth int4 REFERENCES groundwater.Quantity(id) ON DELETE SET NULL,
+	endDepth int4 REFERENCES groundwater.Quantity(id) ON DELETE SET NULL
+);
+
+CREATE TABLE groundwater.gw_geologylog_geologylogcoverage (
+	gw_geologyLog int4 REFERENCES groundwater.GW_GeologyLog(id) ON UPDATE CASCADE ON DELETE CASCADE,
+	result int4 REFERENCES groundwater.GW_GeologyLogCoverage(id) ON UPDATE CASCADE ON DELETE CASCADE
+);
+
+CREATE TABLE groundwater.GW_HydrogeoUnit (
+	id serial PRIMARY KEY
+);
+
+CREATE TABLE groundwater.GW_Well (
+	id serial PRIMARY KEY,
+	gwWellName VARCHAR(50),
+	gwWellLocation geometry(POINT, 4326),
```

Comment:
For all tables where we defined points we should have the following constraints:

```
  CONSTRAINT enforce_dims_wkb_geometry CHECK (st_ndims(wkb_geometry) = 3),
  CONSTRAINT enforce_geotype_wkb_geometry CHECK (geometrytype(wkb_geometry) = 'POINT'::text OR wkb_geometry IS NULL),
  CONSTRAINT enforce_srid_wkb_geometry CHECK (st_srid(wkb_geometry) = 3725)
```